persistence-cassandra-4.0/src/main/java/io/stargate/db/cassandra/CassandraPersistenceActivator.java

```diff
@@ -130,6 +130,7 @@ public static Config makeConfig(File baseDir) throws IOException {
     c.storage_port = listenPort;
     c.listen_address = listenAddress;
     c.broadcast_address = broadcastAddress;
+    c.enable_sasi_indexes = true;
```

Comment:
The PR's title mentions SAI, which usually refers to what is covered by [CEP-7](https://cwiki.apache.org/confluence/display/CASSANDRA/CEP-7%3A+Storage+Attached+Index), but the change enables SASI... Is there a typo?

Reply:
Indeed, the typo is in the issue (https://github.com/stargate/stargate/issues/815) and PR's title. It's enabling SASI indexes. Gonna change that.

Reply:
Should this be configurable and `false` by default because it's experimental?

Reply:
We have set up to true on C* 4.0 because now he have custom index and options enabled: https://github.com/stargate/stargate/pull/813

Reply:
`enable_sasi_indexes` appears to be `true` by default in C* 4.0-rc1 OSS now

Reply:
+1 about keeping it false by default on SG. Gonna revert that in this PR. The only reason I switched to true was for testing. Please, see @olim7t attempts here for the context, https://github.com/stargate/stargate/issues/815 🙂

Reply:
We could still have tests enable that option. The easiest way would be to add a `-D` option for that and process it in the activator. Tests, in turn, can specific custom java system properties for the Stargate node under test via `StargateParameters`

Reply:
`PersistenceTest` sub-classes may also be able to use a custom `cassandra.yaml` in a manner similar to `CassandraPersistenceActivatorTest`